dcplib/checksumming_io/checksumming_sink.py

```diff
@@ -1,18 +1,25 @@
 import hashlib
+
 import crcmod
+
 from .s3_etag import S3Etag
 
 """
-A file-like object that computes checksums for the data written to it, discarding the actual data.
+A file-like object that computes checksums for the data written to it, discarding the actual data. Current accepted 
+checksum functions are crc32, crc32c, s3_etag, sha1, and sha256.
 """
 
 
 class ChecksummingSink:
 
-    def __init__(self, write_chunk_size, hash_functions=('crc32c', 'sha1', 'sha256', 's3_etag')):
+    def __init__(self, write_chunk_size, hash_functions=('crc32', 'crc32c', 'sha1', 'sha256', 's3_etag')):
```

Comment:
This may present an issue. If the Upload checksummer uses the default `hash_functions` value (it does https://github.com/HumanCellAtlas/upload-service/blob/master/upload/common/dss_checksums.py#L158), it will now get an extra checksum, which will break it as it check the list of checksums that come back.  You need to either leave the default value of `has_functions` as is, or change the Upload `DssChecksums.ChecksumComputer` to explicitly request `('sha1', 'sha256', 'crc32c', 's3_etag')`.

Reply:
I don't think we should change the default hash_functions since dcplib is shared across all components. Makes more sense to pass them in explicitly within hca cli and server side. We'll have to add in logic on server side to only save 4 of the 5 as tags

Reply:
Done.